boost_library_docs_tracker/tests/test_models.py

```
@@ -0,0 +1,127 @@
+from boost_library_docs_tracker import services
+
+
+# --- BoostDocContent (3+ tests) ---
+
+
+@pytest.mark.django_db
+def test_boost_doc_content_url_unique(boost_doc_content):
+    """BoostDocContent has a unique url constraint."""
+    assert boost_doc_content.url is not None
+    assert boost_doc_content.id is not None
+    with pytest.raises(Exception):
```

Ruff (0.15.4)

[warning] Do not assert blind exception: `Exception`

(B017)